yt_dlp/extractor/patreon.py

```diff
@@ -137,13 +139,24 @@ def _real_extract(self, url):
                     })
 
         if not info.get('url'):
-            embed_url = try_get(attributes, lambda x: x['embed']['url'])
-            if embed_url:
+            # handle Vimeo embeds
+            if attributes.get('embed').get('provider') == 'Vimeo':
+                embed_html = attributes.get('embed').get('html')
+                v_url = self._search_regex(r'''src=(https%3A%2F%2Fplayer\.vimeo\.com.+)%3F''', embed_html, "vimeo url")
                 info.update({
-                    '_type': 'url',
-                    'url': embed_url,
+                    '_type': 'url_transparent',
+                    'url': VimeoIE._smuggle_referrer(compat_urllib_parse_unquote(v_url), 'https://patreon.com'),
+                    'ie_key': 'Vimeo',
                 })
 
+            else:
```

Comment:
```suggestion
            if not info.get('url'):
```

This is how it is done in the other places